packages/block-library/src/button/edit.js

```diff
@@ -127,6 +128,7 @@ function URLPicker( {
 			position="bottom center"
 			onClose={ () => setIsURLPickerOpen( false ) }
 			anchorRef={ anchorRef?.current }
+			toggleRef={ linkToolbarButtonRef?.current }
```

Comment:
When will `linkToolbarButtonRef` not be an object?

Reply:
Also, it seems better to pass the reference instead of its value?

Reply:
I followed how the `anchorRef` is implemented in this situation. It would be confusing that we are passing the actual value for `anchorRef` but we are passing the reference for `toggleRef`. I prefer passing the reference as well, but I wanted to avoid even more confusion.